website/videoBlog/2024-01-17-How-to-Delete-Items-from-Hudi-using-Delta-Streamer-operating-in-UPSERT-Mode-with-Kafka-Avro-MSG-12.mdx

```diff
@@ -0,0 +1,21 @@
+---
+title: "How to Delete Items from Hudi using Delta Streamer operating in UPSERT Mode with Kafka Avro MSG #12"
+last_modified_at: 2024-01-25T16:54:38.964863-07:00
+authors:
+- name: Soumil Shah
+category: blog
+image: /assets/images/video_blogs/2024-01-17-How-to-Delete-Items-from-Hudi-using-Delta-Streamer-operating-in-UPSERT-Mode-with-Kafka-Avro-MSG-12.png
+navigate: "https://www.youtube.com/watch?v=gnhhjkEwr9o"
+tags:
+- guide
+- beginner
+- apache hudi
+- deletes
+- deltastreamer
+- hudi streamer
+- upsert
```

Comment:
@nfarah86  Can we consolidate all tags on write operations to be consistently singular. I see tags for both `delete` and `deletes`. similarly `upsert` and `upserts`. Can we clean these and add a rule in the readme file ?

Reply:
sure- sounds good.